### PR TITLE
[WEB-504] added SP algorithm page

### PR DESCRIPTION
**Pull request by rhyolight**
@cbaranski @lscheinkman The ticket says to stage this somewhere we can include only BAMI updates? Easiest way to do this would be to create a remote `bami` branch, is that ok by you?

Don't merge yet.

### CODE REVIEW

File: packages/numenta.com/pages/spatial-pooler-algorithm/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Spatial Pooler Algorithm
+========================
+
+The Spatial Pooler is one of the main learning algorithms in HTM. While we are working on a standalone chapter for BAMI that will detail the Spatial Pooler, for now you can explore the following resources to learn more:
+
+- **HTM School episodes about the Spatial Pooler** • [HTM School](http://numenta.org/htm-school/) is a video tutorial series that explains concepts of HTM with an emphasis on visualizations and examples.  These episodes talk about the Spatial Pooler and related functions.
```
Comment (lscheinkman) [inline packages/numenta.com/pages/spatial-pooler-algorithm/index.md]: Should we prefer 'https' over 'http' now that it works? For example `[HTM School](https://numenta.org/htm-school/)`

Comment (rhyolight) [inline packages/numenta.com/pages/spatial-pooler-algorithm/index.md]: Sure I'll do that.

**Comment by lscheinkman** [comment]
@rhyolight You dont need to create a separate branch to show the updates, you can use `gh-pages`. Here are the instructions:
https://github.com/numenta/numenta-web#integration

**Comment by rhyolight** [comment]
I followed the instructions, but something ain't right here...

https://rhyolight.github.io/numenta-web/


---------
Matt Taylor
OS Community Flag-Bearer
Numenta

**Comment by lscheinkman** [comment]
you forgot to update the `config.toml` file

**Comment by rhyolight** [comment]
Here you go @cbaranski 

https://rhyolight.github.io/numenta-web/biological-and-machine-intelligence/

**Comment by rhyolight** [comment]
There is a problem with resource locations, looking into it...

**Comment by rhyolight** [comment]
superseded by https://github.com/numenta/numenta-web/pull/202

**Pull request closed without merging**